syntax = "proto3";

package shoppingService;

service Bidirectional {
  rpc GetOrder(stream ClientItemList) returns (stream ServerItemList) {}
}

message Item {
  string name = 1;
}

message ClientItemList {
  repeated Item clientItems = 1;
}

message ServerItemList {
  repeated Item serverItems = 1;
  string timestamp = 2;
}